app/src/main/java/woowacourse/movie/ui/main/movies/MoviesPresenter.kt

```diff
@@ -0,0 +1,36 @@
+package woowacourse.movie.ui.main.movies
+
+import domain.Movies
+import woowacourse.movie.data.model.mapper.MovieMapper
+import woowacourse.movie.data.model.uimodel.AdvertisementUiModel
+import woowacourse.movie.repository.MoviesRepository
+
+class MoviesPresenter(
+    private val view: MoviesContract.View,
+    private var repository: MoviesRepository
```

Comment:
```suggestion
    private val repository: MoviesRepository
```
val을 활용해볼까요? (이하 관련 내용 동일)